lms/services/d2l_api/client.py

```diff
@@ -78,6 +106,17 @@ def group_set_groups(self, org_unit, group_category_id, user_id=None):
 
         return groups
 
+    def course_table_of_contents(self, org_unit) -> List[dict]:
```

Comment:
Keeping this very close to the actual D2L API as we plan to move things around to support folders.